libs/ui/popover/brain/src/lib/brn-popover-trigger.directive.ts

```diff
@@ -7,7 +7,6 @@ import { BrnPopoverComponent } from './brn-popover.component';
 	standalone: true,
 	host: {
 		'[id]': '_id()',
-		'(click)': 'open()',
```

Comment:
Why is this getting removed?

Reply:
I removed this from the host metadata (along with the other directives) because I was seeing an error where open() was being called twice somehow, specifically in the popover trigger. I assumed it was because the inherited directive (`BrnDialogTriggerDirective`) also had the `(click)` event in the host metadata.  

After reverting those changes, it seems like the double call is not happening anymore, so there must have been another issue that I was seeing at the time which has since been cleared up.

Reply:
Ah actually, I am seeing it again now. I had forgot to add `(click)` back to the popover trigger directive. Now when I click the popover trigger, I can see the `open()` method is being called twice.

I recorded a GIF to demonstrate the issue quickly:

![host-click](https://github.com/goetzrobin/spartan/assets/18401956/aaf4b0c6-626c-44a1-a66d-6af87cdd05aa)